src/components/ContentNode/Column.vue

```diff
@@ -0,0 +1,41 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+<template>
+  <div class="column" :style="style">
+    <slot />
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Column',
+  props: {
+    span: {
+      type: Number,
+      default: null,
+    },
+  },
+  computed: {
+    style: ({ span }) => ({ '--col-span': span }),
+  },
+};
+</script>
+
+<style scoped lang='scss'>
+@import 'docc-render/styles/_core.scss';
+
+.column {
+  grid-column: span var(--col-span);
+  min-width: 0;
```

Comment:
Tells the flex/column item to become smaller than it can right now, so it does not overflow. 

If we have images or text that are wider than the column, this will make the column outgrow what it should.